test/browser/step_definitions/errors.js

```diff
@@ -9,5 +9,5 @@ Then("they should see an error page", async function () {
 
   const errorTitle = await errorPage.getErrorTitle();
 
-  expect(errorTitle).to.equal(errorPage.getSomethingWentWrongMessage());
+  expect(errorTitle).to.equal("Sorry, there is a problem with the service");
```

Comment:
The browser tests are using the [Page Object Model pattern](https://martinfowler.com/bliki/PageObject.html) to abstract UI implementation details, using [Playwright's POM](https://playwright.dev/docs/test-pom)

Could you move this text into the error page, using a similar approach to the `getSomethingWentWrongMessage`